src/components/Contact/Chat.js

```diff
@@ -6,7 +6,7 @@ class Chat extends Component {
       <div>
         <a
           id="Chat"
-          href="m.me/breakfasy"
+          href="https://m.me/breakfasy"
```

Comment:
Contact/Chat.js: Messenger link updated to allow for interaction with potential customers.